tests/integration/api/v1/curated_recommendations/test_curated_recommendations.py

```diff
@@ -224,8 +224,8 @@ async def test_curated_recommendations(repeat):
         assert data["surfaceId"] == ScheduledSurfaceId.NEW_TAB_EN_US
 
         corpus_items = data["data"]
-        # assert total of 80 items returned
-        assert len(corpus_items) == 80
+        # assert total of 100 items returned, which is the default limit
```

Comment:
what do you mean by "default limit" here?

Reply:
I'll clarify the comment:
```suggestion
        # assert total of 100 items returned, which is the default number of recommendations returned
```